css/top-stories.css

```diff
@@ -0,0 +1,141 @@
+/* Main Container */
+
+.main-container {
+    width: 100%;
+    max-width: 1440px;
+    height: 100vh;
+    margin: 0 auto;
+}
+
+/* HEADER */
+
+.header {
+    position: sticky;
+    width: 100%;
+    height: 61px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    background-color: #052450;
+    box-shadow: 0 2px 8px rgba(0, 0, 0, 0.15);
+    box-sizing: border-box;
+}
+
+.header-container {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+}
+
+.header-logo {
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 57px;
+}
+.header-text-logo{
+    color: #FFFFFF;
+    margin-left: 40px;
+}
+
+/* SEARCH */
+.search {
+    position: relative;
+    margin-left: 38px;
+}
+
+.search img {
```

Comment:
You've asked if it's written properly, so for me, it looks fine, and my implementation would be the same, but I cannot promise that it's the best solution, since I'm not very well in CSS and HTML, but I'll teach you a lot of Angular stuff))